cli/api/utils.go

```diff
@@ -41,11 +44,19 @@ func GetAgentIP(defaultRPCPort int) string {
 	if options.Endpoint != "" {
 		return options.Endpoint
 	}
-	agentIP, err := utils.GetIPAddress()
-	if err != nil {
-		panic(err)
+
+	var agentIP string
+	var err error
+	for i := 1; i <= networkAccessAttempts; i++ {
+		if agentIP, err = utils.GetIPAddress(); err == nil {
```

Comment:
@JP--I suspected, but feeling very much out of the loop so trying to cover all bases. :-)